### ISSUE TITLE
Graph page should be more discoverable

### ISSUE THREAD

**Issue by silverwind** [go-gitea/gitea#6140]
The [graph](https://try.gitea.io/swoldemi/NervBot/graph) page of a repo seems undiscoverable to me. To my knowledge it is not linked to anywhere from a repo, or is it? Maybe it should be linked to from the activity tab or even get its own tab.

**Comment by hexfs** [comment]
You can find it on [commit history](https://try.gitea.io/swoldemi/NervBot/commits/branch/master), but I also think It should be linked to activity tab.

**Comment by silverwind** [comment]
Indeed there it is, not the place I would expect it to be. I'm thinking maybe do something similar to GitHub where the activity page would have sub-sections where the commit graph is one of them.

**Comment by hexfs** [comment]
Good ideia, but In my opnion It should be better do #4724 first, then implement the graph with sub-sections like you said.

**Comment by jvpernis** [comment]
I second this. In my opinion, a commit graph is the most powerful tool a git GUI can provide. 

The only product that has been able to pull this off right is the [Charts & Graphs](https://marketplace.atlassian.com/apps/1212147/charts-graphs-for-bitbucket?hosting=server&tab=overview) plugin for Bitbucket:

- Incorporated into the commits page, there is no reason it should be separate
- Clean overview of the commits
- Right aligned graph to prevent a big gap between the graph and text
- Labels on the graphs with branch names, even when they're merged (although I think this is only in the legacy graph mode)

I love gitea, but this is the only thing holding me back from switching over as I use this feature a lot, and at the moment it's too hidden for me.

**Comment by VBoucher** [comment]
I just want to say that I agree. **The graph should have it's own tab**. I didn't find it until I saw @silverwind post and @hexfs comment. Going to commit history and trying to find the graph button (which is a white color button on a white color background) is not helpful. It's not visible.

**Comment by silverwind** [comment]
The issue is while valid, is not concrete enough regarding suggestions, closing.

**Comment by peteklapka** [comment]
I would like a reconsideration of this issue please.  The commit graph is incredibly powerful, and after installing gitea I was disappointed that it doesn't have one... because it was so buried I couldn't find it.  I even went as far as download and install SmartGit to get a commit graph view.  By PURE coincidence, I decide to search for "gitea commit graph" and found out that it exists, but to get to it, you have to know to type /graph at the end of your repo URL, OR you have to (for some reason) look at the history of some random file, and only THEN there is a way to access it through the UI.  Yuck.  Please reconsider!  The same "Commit Graph" button shown on the History page of a file could exist happily next to the "Add File" combo box and LOOK!  Suddenly Gitea has a commit graph feature that's super discoverable!

**Comment by lunny** [comment]
Did you mean the button below.

<img width="376" height="283" alt="Image" src="https://github.com/user-attachments/assets/c0662b5a-477c-43a9-986d-c00f2bdbccf4" />

**Comment by silverwind** [comment]
Yes, that "Commit Graph" button. Maybe we should put it in the "Activity" section. It's not a perfect fit but more discoverable. Could even keep the old button.

Also, maybe compress these 2 lines onto 1 line:

<img width="1331" height="99" alt="Image" src="https://github.com/user-attachments/assets/04a3de2f-0074-4b27-9784-74a843b88b8e" />

**Comment by peteklapka** [comment]
Yes.  That’s the one.
